### ISSUE TITLE
Quicktime videos not playing

### ISSUE THREAD

**Issue by jhammock** [EOL/tramea#294]
eg: 
http://eol.org/data_objects/10547237
http://eol.org/data_objects/24176256

The problem may be addressed here: https://support.apple.com/en-us/HT205081

**Comment by AmrMMorad** [comment]
As per gitter chat, Do you think that we need to wait until reharvesting and then check that?

**Comment by JRice** [comment]
No, I think we should look into it. There _are_ some cases where the video is missing—we can wait on reharvesting for those, but there are others (and I think more common) where it's simply not playing, and we should figure that out (though I think as you mentioned in chat, it should be pretty simple to figure out the tag change required).

**Comment by AmrMMorad** [comment]
I've replaced the tag for the quick time only for now and this worked on my local machine. I've added this to a separated branch. Shall I merge this to the master so we can test?
Thanks

**Comment by jhammock** [comment]
I'm in favor of proceeding. We have nothing to lose on these data objects currently :)

**Comment by JRice** [comment]
Sure. I've been asked not to deploy to production until next week, so
there's no rush on this, but we'll do it then.

**Comment by jhammock** [comment]
This is partly working! Videos in resource 233 are playing (eg:http://eol.org/data_objects/10547237). Videos in resource 170 are not (eg: http://eol.org/data_objects/24176256)

Any ideas?
